EmailValidator/Validation/MultipleValidationWithAnd.php

```diff
@@ -3,23 +3,49 @@
 namespace Egulias\EmailValidator\Validation;
 
 use Egulias\EmailValidator\EmailLexer;
+use Egulias\EmailValidator\Exception\InvalidEmail;
 use Egulias\EmailValidator\Validation\Exception\EmptyValidationList;
 
 class MultipleValidationWithAnd implements EmailValidation
 {
+    /**
+     * @var EmailValidation[]
```

Comment:
Intends to make IDE's analyzer smarter, each elements of `$validations` now can be guessed its type in the loop block like this:

![image](https://cloud.githubusercontent.com/assets/1135118/16621299/b55a440a-43d0-11e6-85db-be83ebd7cfd1.png)